2001-3000/LIT2892RepCh173.xml

```diff
@@ -58,13 +59,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <bibl>
                   <ptr target="bm:Chaine1913Rep"/>
                   <citedRange unit="item">173</citedRange>
+                  <note>Chaine mistakenly attributed this malkǝʾ-hymn to Walatta Ṗeṭros in his Repertoire des Salam, but correctly attributed the hymn to ʾƎḫǝta Krǝstos in <bibl>
```

Comment:
I think "attributed" can be misunderstood as referring to the authorship of the malkǝʾ. Maybe "Chaîne mistakenly identified this as a malkǝʾ-hymn for Walatta Ṗeṭros"?

It would also be possible to use IDs for Chaîne and the Repertoire.